mautic/mautic
# Add Guadeloupe and Saint Pierre and Miquelon

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | #2079
| BC breaks? | N
| Deprecations? | N

#### Description:

This pull request simply adds Guadeloupe and Saint Pierre and Miquelon to the countries list

#### Steps to test this PR:
1. Go to any countries list in Mautic
2. Check that Guadeloupe and Saint Pierre and Miquelon are available
